internal/util/errors.go

```diff
@@ -21,6 +21,13 @@ import (
 	"fmt"
 )
 
+const (
+	// VolumeAlreadyResizedFmt string format to return when FS size is already done.
+	VolumeAlreadyResizedFmt = "FS at the path %s is already resized"
```

Comment:
error messages we return to user should not be capitalized. as these errors are used only in one place better to return error directly from the function